starter-code/index.html

```diff
@@ -2,30 +2,55 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link type="text/css" rel="stylesheet" href="css/main.css"/>
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+<div class="container">
+      <section id="left"><img src="images/phones.png"/></section>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
 
-  Oh yeah, did we mention it's free?
 
-  Download on the App Store
-  Get it on Google Play
+ <section id="right">
+    <section id="miniHeader">
+      <img src="images/brand.png"/>
+      <button class="homeButton">
+        <img src="images/home.png"/>Log In
+      </button>
+    </section>
+    <section id="text">
+  <div>
+  <h2 id="insta">Capture and Share <br/>
+     the World's Moments</h2>
+  <p>
+  Instagram is a fast, beautiful and fun way to share your life with friends and family.<br/>
+  <br/>
+  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy.
+  You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</br>
+  <br/>
+  Oh yeah, did we mention it's free?
+  <p>
+  <div class="app">
+    <img src="images/badge-iphone.png"/>
+    <img src="images/badge-android.png"/>
+  </div>
+  </div>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
+  </section>
+</section>
+<section id="footer">
```

Comment:
You could use the footer tag ( https://www.w3schools.com/tags/tag_footer.asp ), so the you don't need to specify an id for the section